credoai/utils/model_utils.py

```diff
@@ -17,10 +19,28 @@ def get_generic_classifier():
             model = RandomForestClassifier()
         return model
 
+
+def get_model_info(model):
```

Comment:
For Keras and Torch, we can probably infer whether it is classification or regression through looking at the labels in the dataset.

Reply:
Unfortunately, that's a function of the data and the model. We actually may need the objective function - a 10 label dataset could be viewed as an ordinal regression or a classification.